tests/Aspire.Microsoft.EntityFrameworkCore.SqlServer.Tests/Aspire.Microsoft.EntityFrameworkCore.SqlServer.Tests.csproj

```diff
@@ -3,6 +3,9 @@
   <PropertyGroup>
     <TargetFramework>$(NetCurrent)</TargetFramework>
     <RunTestsOnHelix>true</RunTestsOnHelix>
+
+    <!-- ActiveIssue: https://github.com/dotnet/aspire/issues/5057 -->
```

Comment:
I think we also need it for Aspire.Microsoft.Data.SqlClient.Tests as those are also failing.